### PR TITLE
Removed run() options (never used) and ineffective event data dup()

### PR DESCRIPTION
**Pull request by portertech**

### CODE REVIEW

File: lib/sensu/extension.rb [hunk 1]
```diff
@@ -97,27 +95,28 @@ def has_key?(key)
...
-      def safe_run(data=nil, options={})
+      def safe_run(data=nil)
         begin
           @run_arity ||= method(:run).arity.abs
-          arguments = []
-          arguments << (data ? data.dup : data) if @run_arity >= 1
-          arguments << options if @run_arity == 2
-          run(*arguments) do |output, status|
-            yield(output, status)
+          if @run_arity >= 1
```
Comment (portertech) [inline lib/sensu/extension.rb]: KISS

File: lib/sensu/extension.rb [hunk 2]
```diff
@@ -97,27 +95,28 @@ def has_key?(key)
...
       #   expecting to be called with two parameters, an output string
       #   and exit status code.
       # @yieldparam output [String]
       # @yieldparam status [Integer]
-      def safe_run(data=nil, options={})
+      def safe_run(data=nil)
         begin
           @run_arity ||= method(:run).arity.abs
-          arguments = []
-          arguments << (data ? data.dup : data) if @run_arity >= 1
```
Comment (portertech) [inline lib/sensu/extension.rb]: dup() only ever applied to top level hash object :-1:

**Comment by amdprophet** [comment]
:+1:

**Pull request merged**